src/app/onion/learning-object-builder/components/content-upload/app/upload/url-manager/title-of-url/title-of-url.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  Output,
+  EventEmitter,
+  OnDestroy
+} from '@angular/core';
+import { UrlManagerComponent } from '../url-manager.component';
+import 'rxjs/add/operator/takeUntil';
+import 'rxjs/add/operator/debounceTime';
+import { Subject } from 'rxjs/Subject';
```

Comment:
It doesn't appear that we're using anything from RxJS in this component, can we remove all related imports?